4.5/windows/windowsservercore/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM microsoft/windowsservercore
+
+ENV NPM_CONFIG_LOGLEVEL info
+ENV NODE_VERSION 4.5.0
+ENV NODE_SHA256 16aab15b29e79746d1bae708f6a5dbed8ef3c87426a9408f7261163d0cda0f56
+
+RUN powershell -Command \
```

Comment:
I think the biggest benefit of setting the `SHELL` to `powershell` is that it avoids the unexpected pre-processing of the `RUN` lines by `cmd`, which is the only reason `%NODE_VERSION%` works.  Having one shell interpret the line before it is sent to another shell is one situation I would rather avoid.